druid-shell/src/platform/windows/util.rs

```diff
@@ -127,6 +129,15 @@ pub(crate) fn recti_to_rect(rect: RECT) -> Rect {
     )
 }
 
+/// Converts a `Region` into a vec of winapi `RECT`, with a scaling applied.
+pub(crate) fn region_to_rectis(region: &Region, scale: Scale) -> Vec<RECT> {
```

Comment:
"rectis" seems an odd name.

Reply:
I'm not in love with it either, but I copied it from `rect_to_rectis`...